src/components/molecules/Settings/Logo/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Icon from "@reearth/components/atoms/Icon";
+import { styled } from "@reearth/theme";
+
+export interface Props {
+  className?: string;
+}
+
+const Logo: React.FC<Props> = ({ className }) => {
```

Comment:
Is this component used in multiple places? If not, I am not sure it is necessary to make this its own component. Let me know what you think

Reply:
ok, I just think it will be easy. I will remove the component